tests/acceptance/features/apiGraph/addUserToGroup.feature

```diff
@@ -149,3 +149,44 @@ Feature: add users to group
   Scenario: admin tries to add user to a group without sending the group
     When the administrator tries to add user "Alice" to group "" using the Graph API
     Then the HTTP status code should be "404"
+
+
+  Scenario: add multiple users to a group at once
+    Given the administrator has given "Alice" the role "Admin" using the settings api
+    And these users have been created with default attributes and without skeleton files:
+      | username |
+      | Brian    |
+      | Carol    |
+    And user "Alice" creates a group "grp1" using the Graph API
+    When the administrator "Alice" adds the following users to a group "grp1" at once using the Graph API
+      | username |
+      | Brian    |
+      | Carol    |
+    Then the HTTP status code should be "204"
+    And the following users should be listed in the following groups
+      | username | groupname |
+      | Brian    | grp1      |
+      | Carol    | grp1      |
+
+
+  Scenario: admin tries to add users at once to a group without sending the group
+    Given the administrator has given "Alice" the role "Admin" using the settings api
+    And these users have been created with default attributes and without skeleton files:
+      | username |
+      | Brian    |
+      | Carol    |
+    When the administrator "Alice" adds the following users to a group "" at once using the Graph API
+      | username |
+      | Brian    |
+      | Carol    |
+    Then the HTTP status code should be "404"
+
+
+  Scenario: admin tries to add multiple non-existing at once user to a group
```

Comment:
```suggestion
  Scenario: admin tries to add multiple non-existing users at once to a group
  OR
  Scenario: admin tries to add multiple non-existing users to a group at once
```
A small question, what happens when a combination of existing and non-existing users is given? Would you be willing to add scenarios for such edge cases?